src/usecase/get_profile.go

```diff
@@ -0,0 +1,45 @@
+package usecase
+
+import "github.com/aridome222/Backend-Festival-Booth/domain"
+
+type GetProfileUseCase struct {
+	repo domain.ProfileRepository
```

Comment:
```suggestion
	Repo domain.ProfileRepository
```
ここはアッパーキャメルじゃなくてもよいのでしょうか？

Reply:
小文字だと指定パッケージだけで使える、protect になるとのこと回頭いただきました🙇